### DISCUSSION TITLE
What is zettelkasten?

### DISCUSSION THREAD

**Discussion by thany** [Zettlr/Zettlr#5051] [General]
Since Zettlr takes pride in that it's based on the ideas and methodology of zettelkasten - what actually _is_ that?

On zettelkasten.de I see a enormous wall of blahblah text, but not just a single concise paragraph on what makes it allegedly so brilliant.

Maybe on the homepage or something, spend just one or two paragraphs on what this zettelkasten is exactly, and why it's so miraculously different from, say, a folder.

What makes it unique? What makes it brilliant? What other methodology is it similar but superior to?

**Comment by lordofscripts** [comment]
I know how you feel, last week I was trying to start writing my eBook and was considering using the VSCode Hyperbook but the site was full of blah, blah but leaving you in total obscurity about many things. I had to move on and seek another tool.

**Comment by hittjwX** [comment]
Think of Zettelkasten as a stack of index cards on a corkboard, connected together by colored strings. You're working on your paper. An idea comes up. You put it on a card (a new brief note), then pin it to the board.

The card has a title, description, and index. You might have had similar ideas in the past, so you reference them. Maybe it is related to a paper you are writing, so you use a hashtag that represents that project.

As you are working on your paper, build out new cards from research. Or assemble cards in a certain order. Maybe group those cards into chapters, represented in a content map. (MOC) On the corkboard, these might be columns to stack cards.

In Zettlr, that note can be created with Ctrl-N, given a filename that contains a serial number, then referenced from everywhere. Links representing relationships can be made with `[[FILENAME]]` references. You don't need to physically move around cards.

Those cards can be stacked, shuffled, searched, referenced, or digested into your main paper without much effort. The MOC represents your paper, pointing to relevant cards.

The paper presents ideas derived from a Zettelkasten ecosystem.

Because those cards are single topics, they might reference several papers, citations, other notes, or cards. Overall, creating a graph helps you discover new ideas. Think atomic notes.

It's different from folders because the cards can be anywhere; the reference creates structure while organizing ideas. The deliverable is your paper, not the note-taking itself.

If this sounds like walls covered with cards and strings, think of your paper as a specific color string threading notes into a deliverable. You'll see things in those other threads, or connections that make it easier to extract useful ideas
